ayeniblessing101/Post_It
# #152627280 feedback implementation 152627280

#### What does this PR do?
Implement all feedback from scorecard
#### Description of Task to be completed?
  ####System Functions
  - Fix On successful signup, when the user clicks on the "POST IT" logo, the user is taken to the signup page.
- Fix Signed in users can access the login page even after they have been logged in.
- Fix Error message on  signin form
- Ensure field doesn’t clear out when there is an error on signup and signing.
- Review the password-reset feature that seem inconsistent
-Fix notification
  ####Task Runner
- Fix webpack for production build

 ####Project Management & Workflow
- remove file that need to be ignored
- Populate the "Limitations", "FAQ" section of my repo
- "localhost:3000" has been properly formatted in my repo readme and include the git command for cloning in the getting started section
- Added a "Contributions" sections

  ####Test Coverage
- Ensure that there are no eslint violations on your test files.
- Ensure that my coverage is not below 80% 

  ####Code Quality
- Made all filenames for in server side camelCase.
- Proper file naming convention
-fix pagination
- Provided proper JsDocs for yours helpers and also your middleware, controllers, actions. validations, 
- Fix improper lines spacing in your app.js file and every where necessary
-Remove all commented code.

   ####UI/UX
- The register button seems to do just nothing. Is that the idea behind the button.
- User is able to upload image
- Rename "Group" on the header to "Menu" 
- Apply some padding to placeholder/content in the search bar.
- In responsive view,  fix overlay covering the entire screen.
- Made search bar and Welcome text appears in responsive view.
- Add titles for "Groups" and "Users" and then reposition "Add Users" button to provide a better UI for this application Which made "Send/post" button more visible
 This will also make "Send/post" button for the message board more visible.
- Apply some spacing between the elements in header
- Add a tooltip that says "Logout" such that it more descriptive on the logout icon.

#### How should this be manually tested?
- Please refer to repo
- blessing-post-it.herokuapp.com
#### Any background context you want to provide?
NA
#### What are the relevant pivotal tracker stories?
#152627280
#### Screenshots (if appropriate)
NA
#### Questions:
NA
